sass/includes/_alert-message.scss

```diff
@@ -25,13 +25,19 @@
         padding-left: 0.4rem;
         padding-right: 0.4rem;
         margin-right: 0.4rem;
+        font-weight: bold;
+        text-transform: uppercase;
     }
 
     &__description {
         margin-bottom: 0;
         display: inline-block;
     }
 
+    &__description a {
```

Comment:
Would you be able to remove this, as the styling  for the `<a>` should be nested inside main `&__description` class.